src/eslint-config-adeira/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Remove `__` global, legacy of Nitrolib from Kiwi.com, see: https://kiwicom.github.io/nitrolib/services.html#intl
+- Set `flowtype/require-readonly-react-props` to NEXT_VERSION_ERROR
```

Comment:
`NEXT_VERSION_ERROR` is quite an internal thingy. I usually write something like "warnings or errors in strict mode" for the public. Would you please consider it?

Reply:
Yes, you are right, I will update it

Reply:
🚀